tests/cockroachdb_import_connect.robot

```diff
@@ -40,3 +40,10 @@ Scenario: Connect CockroachDB DBSC With An Openshift Application
     And User Imports Openshift cockroach Application From YAML
     And User Creates Service Binding Between cockroach DBSC Instance And Imported Openshift Application
     Then The Application Accesses The Connected cockroach Database Instance
+
+Scenario: Import CockroachDB Provider Account From Developer View
+    [Tags]    smoke    RHOD-180
+    When User Navigates To Add CockroachDB Topology Screen From Developer View
+    And User Navigates To Import Provider Account Screen From Developer View
+    And User Enters Data To Import CockroachDB Provider Account
+    Then Provider Account Import Success
```

Comment:
@ikanias can we please move this test case to the very top, the prerequisite for this to work is that we do not have any provider accounts imported, but when we execute this test suite and reach this test case, our cluster will already have an invalid provider account imported from the test case -_Scenario: Verify error message for invalid credentials on CockroachDB_ and therefore we will not see the import a provider account link on the developers add to topology page.

Our test and suite teardown will also only remove the valid provider accounts and therefore the existence of an imported invalid provider account may cause problems for this test case.

Reply:
I agree on this